tests/device_tests/test_msg_signtx.py

```diff
@@ -621,21 +686,62 @@ def test_fee_too_high(self, client):
             == "c36928aca6452d50cb63e2592200bbcc3722ce6b631b1dfd185ccdf9a954af28"
         )
 
+    @pytest.mark.skip_t1
+    def test_fee_high_hardfail(self, client):
```

Comment:
i don't think we need to test mainnet and testnet separately, what do you think @tsusanka ? one of the two should be enough

Reply:
Yeah I agree. Let's remove that

Reply:
Removed `test_testnet_fee_high_hardfail` in 52b45027.